trade-finance-manager-api/src/constants/deals.js

```diff
@@ -58,6 +58,22 @@ const TFM_SORT_BY = {
   DESCENDING: 'descending',
 };
 
+const AMENDMENT_UW_DECISION = {
+  APPROVED_WITH_CONDITIONS: 'Approved with conditions',
+  APPROVED_WITHOUT_CONDITIONS: 'Approved without conditions',
+  DECLINED: 'Declined',
+};
+
+const AMENDMENT_TYPE = {
```

Comment:
where are we using this?

Reply:
being used for which has been approved and declined.  Used a constant as compared to hard coding it